toucan_connectors/google_big_query/google_big_query_connector.py

```diff
@@ -169,7 +169,9 @@ def _prepare_parameters(query: str, parameters: Optional[Dict]) -> List:
     @staticmethod
     def _prepare_query(query: str) -> str:
         """replace ToucanToco variable definition by Google Big Query variable definition"""
-        new_query = query.replace('{{', '@').replace('}}', '')
+        new_query = (
+            query.replace('{{ ', '{{').replace('{{', '@').replace(' }}', '}}').replace('}}', '')
+        )
```

Comment:
There is a helper function for the interpolation named : sanitize_query.
You can have a example here 
https://github.com/ToucanToco/toucan-connectors/blob/master/toucan_connectors/awsathena/awsathena_connector.py#L58